packages/core/src/classes/expression/expression-input-parser.ts

```diff
@@ -182,13 +182,13 @@ export class ExpressionInputParser {
             const [operator, operatorValue] = Object.entries(
               value as any
             )[0] as [
-              (
-                | UpdateType.ArithmeticOperator
-                | UpdateType.SetUpdateOperator
-                | UpdateType.Action
-              ),
-              any
-            ];
+                (
```

Comment:
@andygonzalez2010 
Please remove whitespace changes., may be format the file before pushing the changes up.